shap/explainers/_tree.py

```diff
@@ -1196,7 +1196,7 @@ def buildTree(index, node):
                     self.values[index] = [node.prediction()] #prediction for the node
                 else:
                     self.values[index] = [e for e in node.impurityStats().stats()] #for gini: NDarray(numLabel): 1 per label: number of item for each label which went through this node
-                self.node_sample_weight[index] = node.impurityStats().count() #weighted count of element trough this node
+                self.node_sample_weight[index] = sum([e for e in node.impurityStats().stats()]) #weighted count of element trough this node
```

Comment:
Could you please explain why this is the correct way to do things? I am not deeply familiar with this code.

Reply:
You cam find the answer above.

Reply:
It is because
```bash
 node.impurityStats().count()
```
rounds the values in PySpark models, so it return zero values when it was not zero values

Reply:
If you can please merge this PR. Thanks.